cmd/main.go

```diff
@@ -57,7 +56,7 @@ func buildScheme() (err error) {
 
 //
 // addonManager
-func addonManager(db *gorm.DB, adminChanged chan int) (mgr manager.Manager, err error) {
+func addonManager(db *gorm.DB) (mgr manager.Manager, err error) {
```

Comment:
The addon changed was a channel used to signal to the addon controller that a volume was added/removed/changed.
The addon controller no longer needs to do this because volumes no longer exist.